src/test/components/Root-history.test.js

```diff
@@ -24,6 +24,12 @@ import type { SerializableProfile } from 'firefox-profiler/types';
 jest.mock('../../components/app/ListOfPublishedProfiles', () => ({
   ListOfPublishedProfiles: 'list-of-published-profiles',
 }));
+//AppLocalizationProvider calls a async function in componentDidMount() that uses the following function.
+//The tests related to it are in separate file.
+jest.mock('../../app-logic/l10n-bundling', () => ({
+  ...jest.requireActual('../../app-logic/l10n-bundling'),
+  fetchMessages: jest.fn().mockResolvedValue(['en-US', 'message']),
+}));
```

Comment:
While we were discussing this on chat, Julien suggested to mock the `AppLocalizationProvider` instead like the following code. It would be good to if you can change this as well.
It should look like this:
```js
jest.mock('../../components/app/AppLocalizationProvider', () => ({
  AppLocalizationProvider: ({ children }) => children,
}));
```
It's better to actually mock the component instead of the app-logic because it's better to mock the component itself instead of inner logic. So that way inner logic can change and we will not have to update this code.